paddle/fluid/distributed/auto_parallel/spmd_rules/common.cc

```diff
@@ -0,0 +1,212 @@
+/* Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include <glog/logging.h>
+
+#include "paddle/fluid/distributed/auto_parallel/spmd_rules/common.h"
+#include "paddle/fluid/distributed/auto_parallel/spmd_rules/rules.h"
```

Comment:
```suggestion
#include "paddle/fluid/distributed/auto_parallel/spmd_rules/common.h"

#include <glog/logging.h>

#include "paddle/fluid/distributed/auto_parallel/spmd_rules/rules.h"
```
Put the related header first.